test/data/test_multi_embedding_tensor.py

```diff
@@ -117,3 +118,138 @@ def test_clone():
     assert met_clone.values[0, 0] != 12345.
     met.offset[0] = -1
     assert met_clone.offset[0] != -1
+
+
+def test_cat():
```

Comment:
I think this `cat`'s test is good, but a more direct way is some test like [this](https://github.com/pyg-team/pytorch-frame/blob/master/test/data/test_multi_nested_tensor.py#L164-L178). This requires slicing to be first implemented. Not a blocker, but I'd prefer swapping this `test_cat` with something like [this](https://github.com/pyg-team/pytorch-frame/blob/master/test/data/test_multi_nested_tensor.py#L164-L178) in the subsequent PRs (once slicing is done and tested.)

Reply:
My main concern with this testing approach is that the comparison in offset and values space is quite unintuitive.

Reply:
I see, that totally makes sense :) I'll make these test cases consistent with the `MultiNestedTensor` ones in a follow-up PR adding slicing!

Reply:
Sounds good, thanks! the doc-string of `cat` can be also modified. We try not to expose `values` and `offset` since they are for internal storage.